evalml/data_checks/outliers_data_check.py

```diff
@@ -56,11 +55,15 @@ def get_IQR(df, k=2.0):
             upper_bound = q3 + (k * iqr)
             return (lower_bound, upper_bound)
 
-        clf = IsolationForest(random_state=self.random_state)
-        clf.fit(X)
-        scores = pd.Series(clf.decision_function(X))
-        lower_bound, upper_bound = get_IQR(scores, k=2)
-        outliers = (scores < lower_bound) | (scores > upper_bound)
-        outliers_indices = outliers[outliers].index.values.tolist()
-        warning_msg = "Row '{}' is likely to have outlier data"
-        return [DataCheckWarning(warning_msg.format(row_index), self.name) for row_index in outliers_indices]
+        lower_bound, upper_bound = get_IQR(X)
+        indices = set()
+        # get the columns that fall out of the bounds, which means they contain outliers
+        for idx, bound in enumerate([lower_bound, upper_bound]):
```

Comment:
This was a little hard to follow at first: maybe an approach like [this](https://stackoverflow.com/questions/40156469/how-to-check-if-any-value-of-a-column-is-in-a-range-in-between-two-values-in-p) (but check for outside of range and map to every column in the DF).